azurerm/internal/services/iothub/validate/digital_twins_test.go

```diff
@@ -0,0 +1,59 @@
+package validate
+
+import (
+	"testing"
+)
+
+func TestDigitaltwinsName(t *testing.T) {
```

Comment:
Can we add two more test cases here? One for exact length (63 chars) and one for a name that is too long (64 chars)?